src/screens/JobRun/JobRunView.tsx

```diff
@@ -78,6 +86,18 @@ export const JobRunView = ({ run }: Props) => {
     }
   }, {})
 
+  //When there are no errors, taskRun is empty so attributes are not set, we
+  //must insert attributes
+  if (run.allErrors.length == 0) {
+    const graph = parseDot(`digraph {${run.job.observationSource}}`)
+    graph.forEach((node) => {
+      attrs[node['id']] = {
+        ...node.attributes,
+        status: TaskRunStatus.COMPLETE,
+      }
```

Comment:
Object property access is typically done with dot `.` notation, bracket notation is saved for string expressions when the property name isnt a valid JS identifier 

```suggestion
    graph.forEach((node) => {
      attrs[node.id] = {
        ...node.attributes,
        status: TaskRunStatus.COMPLETE,
      }
```

Reply:
Fixed in [`02f5f96` (#29)](https://github.com/smartcontractkit/operator-ui/pull/29/commits/02f5f965002de39bbae9c13b1de9315673725661)